### PR TITLE
Override `crypto-js` resolution

### LINKED ISSUE #9472: Handle transitive dependencies override consistently
I believe we should use `resolutions` or the `overrides` field in `package.json` rather than manually modifying the `yarn.lock` file.

We currently want to force `crypto-js` to use **4.2** instead of **^3.3**. This seem to break the [Snyk workflow on `main`](https://github.com/guardian/dotcom-rendering/actions/runs/6862330146/job/18660217546).

Related work:

- #9457
- #9339 
- #9461 
- #9462

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Ensure that all versions of `crypto-js` use **4.2.0**

https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/
https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md

## Why?

Resolves #9472 

## Screenshots

N/A

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -15203,7 +15203,7 @@ preact@10.15.1:
     core-js "^3.13.0"
     core-js-pure "^3.13.0"
     criteo-direct-rsa-validate "^1.1.0"
-    crypto-js "^3.3.0"
+    crypto-js "^4.2.0"
```
Comment (mxdvl) [inline yarn.lock:15206]: This was done automatically by `yarn`

File: yarn.lock [hunk 2]
```diff
@@ -15191,7 +15191,7 @@ preact@10.15.1:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.15.1.tgz#a1de60c9fc0c79a522d969c65dcaddc5d994eede"
   integrity sha512-qs2ansoQEwzNiV5eAcRT1p1EC/dmEzaATVDJNiB3g2sRDWdA7b7MurXdJjB2+/WQktGWZwxvDrnuRFbWuIr64g==
 
-"prebid.js@github:guardian/prebid.js#0b4cccd55ab4e170b36bb944e9bdd2807a9da1c5":
+prebid.js@guardian/prebid.js#0b4cccd55ab4e170b36bb944e9bdd2807a9da1c5:
```
Comment (ioannakok) [inline yarn.lock:15194]: Is this change intended?

Comment (arelra) [reply yarn.lock:15194]: This is the correct version i.e. without the 'github' prefix - so all good

**Review by arelra — APPROVED** [review]
